mail_environment/views/fetchmail_server_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <record model="ir.ui.view" id="inherit_fetchmail">
```

Comment:
A record's ID should be placed before the model, according to [this](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#security-view-and-action)